pkg/network/driver/handle.go

```diff
@@ -12,11 +12,10 @@ import (
 	"fmt"
 	"unsafe"
 
-	nettelemetry "github.com/DataDog/datadog-agent/pkg/network/telemetry"
-	"github.com/DataDog/datadog-agent/pkg/telemetry"
-
 	"golang.org/x/sys/windows"
 
+	ebpftelemetry "github.com/DataDog/datadog-agent/pkg/ebpf/telemetry"
```

Comment:
why do we have **ebpf** telemetry in a windows file?

Reply:
moved the relevant files under the general `/pkg/telemetry` to avoid this unnecessary dependency